libr/asm/p/asm_x86_nz.c

```diff
@@ -5113,10 +5113,11 @@ static int oprep(RAsm *a, ut8 *data, const Opcode *op) {
 					free (instr.mnemonic);
 					return -1;
 				}
-				ut8 *ptr = (ut8 *)&lt_ptr->opcode;
-				int i = 0;
-				for (; i < lt_ptr->size; i++) {
-					data[i + l] = ptr[lt_ptr->size - (i + 1)];
+				ut64 opcode = lt_ptr->opcode;
+				int i = lt_ptr->size - 1;
```

Comment:
Sizet. Shifting on signed vars is undefined behaviour

Reply:
It's an unsigned variable, no undefined behavior I suppose.

Reply:
Int is signed

Reply:
yes, but the shift isn’t operated on i.

Reply:
i do not see anything wrong with this. shift occurs only on `opcode` which is unsigned64.